assets/css/src/components-plugin/_breadcrumb.scss

```diff
@@ -9,10 +9,10 @@
 }
 
 .mailpoet_breadcrumb a {
-  color: #444;
+  color: $color-text;
   text-decoration: none;
 }
 
 .mailpoet_breadcrumb a:hover {
-  color: darken(#444, 50%);
+  color: darken($color-text, 50%);
```

Comment:
I wonder if we should keep all the colours in the settings and use variables everywhere else. If we use functions like `darken` only in settings we will have ALL colours defined in one place. If we spread them everywhere we will have no idea how many colours we actually use.